### ISSUE TITLE
Lyrics doesn't match

### ISSUE THREAD

**Issue by kaankaymaz6** [ciderapp/Cider-2#434]
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://cider.sh/legal/conduct) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/ciderapp/Sabiiro-BugTracker/issues) for a bug report that matches the one I want to file, without success.
- [X] I have read the [troublshooting FAQs](https://cider.gitbook.io/welcome-to-gitbook/support/faqs) and none solved my issue.

### What operating system are you using?

Windows

### Build Details

Song lyrics doesn't match

### Operating System Version

Windows 10

### Describe the Bug

While the lyrics are followed instantly, the next line appears as the lyrics of the song before the current song. Because of that, lyrics get mixed and it makes it hard to follow the lyrics. 

### Steps to Reproduce

1. Click on the lyrics button from the right upper corner. It's the second option.
2. Play the song then play another song
3. Lyrics that are on the right will be belong to the previous song.

**Comment by coredev-uk** [comment]
This issue has been closed to consolidate the issue tracker for Cider 2.3.1 and later issues. If you believe this issue is still relevant, please open a new issue.
